### DISCUSSION TITLE
No internet Connection Bridged Mode

### DISCUSSION THREAD

**Discussion by JoaoMSacramento** [utmapp/UTM#6801] [Linux]
**Describe the issue**  
I had my Ubuntu 24.04.1 LTS running fine on UTM with Shared Network as Network Mode. However, for a work, I needed to change the Network Mode to bridge, what I did, and after start the VM I had no internet connection.
Tried to change it back to Shared Network but there was no internet connection still.

My Network configuration right now is:
**Network Mode:** Bridged (Advanced)
**Bridged Network:** en0
**Emulated Network Card:** virtio-net-pci

Running `ifconfig` returns:
```
lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 1338  bytes 106154 (106.1 KB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 1338  bytes 106154 (106.1 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
 
```

Running `nmcli` returns:
```
lo: connected (externally) to lo
        "lo"
        loopback (unknown), 00:00:00:00:00:00, sw, mtu 65536
        inet4 127.0.0.1/8
        inet6 ::1/128

enp0s1: unmanaged
        "Red Hat Virtio"
        ethernet (virtio_net), 9A:7E:04:49:F7:4E, hw, mtu 1500

Use "nmcli device show" to get complete information about known devices and
"nmcli connection show" to get an overview on active connection profiles.

Consult nmcli(1) and nmcli-examples(7) manual pages for complete usage details.


```
**Configuration**  
* UTM Version: 4.4.3 (91)
* macOS Version: 15.1 (24B83)
* Mac Chip (Intel, M1, ...): M2

**Debug log**  
[debug.log](https://github.com/user-attachments/files/17618998/debug.log)

**Upload VM**  
[config.plist.zip](https://github.com/user-attachments/files/17619012/config.plist.zip)

**Comment by hilalyoney** [comment]
I am also experiencing this problem. If it has been resolved, can you please tell me? There is an explanation on UTM's own website regarding the issue as follows. Bridged Interface
Only applicable for “bridged” network mode. Note that bridging with a Wifi interface may require additional configuration. But there is no detail regarding the configuration.

**Comment by zq1040657944** [comment]
I also encountered the same problem. Bridge mode cannot obtain network card information. My environment is macOS

**Comment by SeibertBr** [comment]
impossible to get internet connection.

running win11 on vm.

sequoia 15.3.1
utm Version 4.6.4 (107)


I've tried all the possibilities.. all the network options on utm.. bridge, emulated, etc.. 
also downloaded the latest spice version and installed it in the vm. no difference.

can anyone please share some more ideas on how to solve it? I need to have internet connection to be able to run an erp system inside the vm.
